src/index/ivf/ivf.cc

```diff
@@ -45,12 +46,13 @@ struct QuantizerT<faiss::IndexBinaryIVF> {
 template <typename T>
 class IvfIndexNode : public IndexNode {
  public:
-    IvfIndexNode(const std::string& /*version*/, const Object& object) : index_(nullptr) {
+    IvfIndexNode(const std::string& version, const Object& object) : index_(nullptr) {
         static_assert(std::is_same<T, faiss::IndexIVFFlat>::value || std::is_same<T, faiss::IndexIVFFlatCC>::value ||
                           std::is_same<T, faiss::IndexIVFPQ>::value ||
                           std::is_same<T, faiss::IndexIVFScalarQuantizer>::value ||
                           std::is_same<T, faiss::IndexBinaryIVF>::value || std::is_same<T, faiss::IndexScaNN>::value,
                       "not support");
+        versoin_ = version;
```

Comment:
1. we can store a `Version` object instead of a string in the field. 
2. Feel free to put this `Version` object into `IndexNode`'s field.

Reply:
updated